From: Dennis Braun <d_braun@kabelmail.de>
Date: Sat, 10 Sep 2022 22:14:36 +0200
Subject: Fix build with portsmf system library

Forwarded: not-yet
---
 lib-src/header-substitutes/allegro.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/lib-src/header-substitutes/allegro.h
+++ b/lib-src/header-substitutes/allegro.h
@@ -7,16 +7,16 @@
 
 #pragma warning( push )
 #pragma warning( disable : 4100)
-#include "../portsmf/allegro.h"
+#include <portSMF/allegro.h>
 #pragma warning( pop )
 
 #elif defined(__linux__)
 
 #include <cstring> // Allegro include fails if this header isn't included due to no memcpy
-#include "../portsmf/allegro.h"
+#include <portSMF/allegro.h>
 
 #else //_MSC_VER
 
-#include "../portsmf/allegro.h"
+#include <portSMF/allegro.h>
 
 #endif //_MSC_VER
