--- a/libraries/lib-time-and-pitch/StaffPad/pffft/pfsimd_macros.h
+++ b/libraries/lib-time-and-pitch/StaffPad/pffft/pfsimd_macros.h
@@ -122,7 +122,7 @@
 /*
   ARM NEON support macros
 */
-#elif !defined(PFFFT_SIMD_DISABLE) && (defined(__arm__) || defined(__aarch64__) || defined(__arm64__) || defined(_M_ARM64))
+#elif !defined(PFFFT_SIMD_DISABLE) && (defined(__aarch64__) || defined(__arm64__) || defined(_M_ARM64))
 #  include <arm_neon.h>
 typedef float32x4_t v4sf;
 #  define SIMD_SZ 4
